app/controllers/repairs_controller.rb

```diff
@@ -0,0 +1,35 @@
+class RepairsController < ApplicationController
+    def index
+        @repair = Repair.all 
```

Comment:
Since `Repair.all` returns the list of all of the repairs, it would be best to use `@repairs` (plural).